src/styles/bloks/stars.scss

```diff
@@ -0,0 +1,42 @@
+@use '../utils/variables' as *;
+
+.stars {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  &--4 {
+    margin-left: $card-margin;
```

Comment:
The variable '$card-margin' is used here. Ensure it is defined in the 'variables' file. If not, define it or import it from the correct file.